platform-docs/docs/intro.md

```diff
@@ -33,66 +31,69 @@ We're going to be using JSX to write our UI and components as the block editor i
 
 ## Bootstrap your block editor
 
-It's time to render our first block editor. Update your `src/App.jsx` file with the following code:
+It's time to render our first block editor. We’ll do this with changes to three files – `index.html`, `src/main.jsx`, and `src/App.jsx`.
+
+First, we’ll add the base styles are for the editor UI. In `index.html` add these styles in the `<head>`:
+```html
+<link href="node_modules/@wordpress/components/build-style/style.css" rel="stylesheet" vite-ignore/>
+<link href="node_modules/@wordpress/block-editor/build-style/style.css" rel="stylesheet" vite-ignore/>
+```
+:::note
+
+There are more styles needed but can’t be added here because they are for the editor’s content which is in a separate document within an `<iframe>`. We’ll add those styles via the `BlockCanvas` component in a later step.
+
+:::
+
+Next, we’ll add blocks for the editor to work with. In `src/main.jsx` import and call `registerCoreBlocks`:
+```js
+import { registerCoreBlocks } from '@wordpress/block-library'
+registerCoreBlocks();
+```
+
+Finally, we’ll put our editor together. In `src/App.jsx` replace the contents with the following code:
 
 ```jsx
 import { useState } from "react";
 import {
   BlockEditorProvider,
   BlockCanvas,
 } from "@wordpress/block-editor";
-import { registerCoreBlocks } from "@wordpress/block-library";
-import { getBlockTypes } from "@wordpress/blocks";
 
-// Default styles that are needed for the editor.
+// Base styles for the content within the block canvas iframe.
 import componentsStyles from "@wordpress/components/build-style/style.css?raw";
-import blockEditorStyles from "@wordpress/block-editor/build-style/style.css?raw";
 import blockEditorContentStyles from "@wordpress/block-editor/build-style/content.css?raw";
-
-// Default styles that are needed for the core blocks.
-import blocksCommonStyles from "@wordpress/block-library/build-style/common.css?raw";
```

Comment:
I realized this one was extra, it is `@import`‘d in the block library’s style (the one on the next line).